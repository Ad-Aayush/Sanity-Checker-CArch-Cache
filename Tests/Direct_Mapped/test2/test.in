cache_sim enable ./Tests/Direct_Mapped/test2/config.txt
load ./Tests/Direct_Mapped/test2/input.s
run
cache_sim dump cache.dump

