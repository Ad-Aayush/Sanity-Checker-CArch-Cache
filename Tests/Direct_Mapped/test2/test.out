Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
Executed ... PC=...
D-cache statistics: Accesses=15, Hit=12, Miss=3, Hit Rate=0.80
